test/prototype_transforms_kernel_infos.py

```diff
@@ -1252,7 +1272,9 @@ def sample_inputs_perspective_video():
         KernelInfo(
             F.perspective_video,
             sample_inputs_fn=sample_inputs_perspective_video,
-            closeness_kwargs=cuda_vs_cpu_pixel_difference(),
+            closeness_kwargs={
+                **cuda_vs_cpu_pixel_difference(),
```

Comment:
to revert (note for myself)

Reply:
I don't understand this change or this comment?